src/transformers/models/clip/configuration_clip.py

```diff
@@ -270,7 +270,27 @@ class CLIPConfig(PretrainedConfig):
             The inital value of the *logit_scale* paramter. Default is used as per the original CLIP implementation.
         kwargs (*optional*):
             Dictionary of keyword arguments.
-    """
+
+    Example:
+
+    ```python
+    >>> from transformers import CLIPConfig, CLIPModel
+
+    >>> # Initializing a CLIPConfig with openai/clip-vit-base-patch32 style configuration
+    >>> configuration = CLIPConfig()
+
+    >>> # Initializing a CLIPModel (with random weights) from the openai/clip-vit-base-patch32 style configuration
+    >>> model = CLIPModel(configuration)
+
+    >>> # Accessing the model configuration
+    >>> configuration = model.config
+
```

Comment:
Let's add a comment to explain the objective here, like

`# We can also initialize a CLIPConfig from a CLIPTextConfig and a CLIPVisionConfig`